cloudfoundry/managers/v3appdeployers/standard_strategy_v3.go

```diff
@@ -204,6 +213,8 @@ func (s Standard) Deploy(appDeploy AppDeploy) (AppDeployResponse, error) {
 			},
 			ReversePrevious: defaultReverse,
 		},
+		s.actor.ScaleApplicationProcess(appDeploy, defaultReverse),
+		s.actor.UpdateApplicationProcess(appDeploy, defaultReverse),
```

Comment:
What issue does this fix? Why does it need to scale up here? How did it work before without calling this scale up?

Reply:
This is in place specifically for when we deploy a new application with `stopped = true`. With the current version the provider won't increase the number of processes and won't update health-check-related fields, which causes problems on the 2nd `terraform apply`.

This is only a quick fix though and does not address the whole problem of not properly managing apps deployed in `stopped` state, and we haven't seen any issues since the release of v0.50.0 because it is also a very specific use-case 😄. I'll address this problematic in another PR.

Reply:
Also this won't affect other use-cases since this exact function is called later in the standard deployment flow.